src/OffloadRun.php

```diff
@@ -42,4 +46,36 @@ public function getCacheOptions()
 	{
 		return $this->cache_options;
 	}
+
+	/**
+	 * @return float|null Null to use default TTLs
+	 */
+	public function getTtlFresh()
+	{
+		return $this->ttl_fresh;
+	}
+
+	/**
+	 * @param float|null $ttl_fresh
```

Comment:
Missing description.
`float|null $ttl_fresh $ttl_fresh The TTL to use for the fresh time period (null for default).`